deploy/ui/form.json

```diff
@@ -1180,6 +1180,89 @@
                     }
                   ]
                 },
+                {
+                  "label": "Enable Kafka Extension",
+                  "type": "object",
+                  "max": 1,
+                  "jsonPath": "$.spec.objects.servers[*].kafka",
+                  "visible": true,
+                  "fields": [
+                    {
+                      "label": "Enable Kafka extension",
```

Comment:
@tchughesiv is there a way to make a field invisible?
this one would make sense as the button before explicit "enable" this feature.

Reply:
it would seem so... see line #1188 above...
`"visible": false,`?

Reply:
i do wonder though... is the `extEnabled` field even necessary given the kafa object is a pointer? couldn't we just use that pointer as a means to determine whether to enable? if a user is setting anything in the kafka object, that would mean they want it enabled right?

Reply:
the visible is on a fieldGroup i.e. the button to enable something, not on a single field like checkbox or text

Reply:
is an invisible field this still necessary now that we're using a pointer for template?